packages/notifications/addon/components/navi-notifications.hbs

```diff
@@ -0,0 +1,8 @@
+{{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="navi-notifications" ...attributes>
+  <div>
+    {{#animated-each this.flashMessages.queue use=this.transition as |flash|}}
+      <NaviNotifications::Alert @flash={{flash}} class="m-b-10" />
```

Comment:
Could do `@onClose={{flash.destroyMessage}}`

Reply:
Could be done. Since this is an adapter/wrapper around DenaliAlert for ember-cli-flash (mainly for icon mapping), I didn't think it needed more than exposing a `flash` arg as its public interface. Any strong reasons to expose `onClose` as a first class arg?

Reply:
It just seemed cleaner that the alert wouldn't know about `ember-cli-flash` and that onClose it would pass the action up rather than destroy itself